### PR TITLE
Save push tokens

### PR DESCRIPTION
**Pull request by kruperfone**
Change deploying target to 7.1
Add register for notifications method
Add code signing

### CODE REVIEW

**Comment by vladimirgoncharov** [comment]
:+1:

**Comment by nikitafomin** [comment]
@NikolaevSergey I think we should not setup our certs because this is open repository.

File: Swift-Base/Constants.swift [hunk 1]
```diff
@@ -12,6 +12,10 @@ import UIKit
 let kAPIKeyCrashlitycs  = ""
 let kAPIKeyGoogle       = ""
 
+/*--------------User Defaults keys-------------*/
+let kUserDefaultsDeviceTokenData = "kUserDefaultsDeviceTokenData"
+let kUserDefaultsDeviceTokenString = "kUserDefaultsDeviceTokenString"
+
 /*----------Notifications---------*/
 let kMockNotification:String = "mock_notification";
```
Comment (nikitafomin) [inline Swift-Base/Constants.swift:20]: @NikolaevSergey @VladimirGoncharov can we use names without `k` prefix for notification names? Maybe we should replace `kMockNotification` with `MockNotification`. I use `k` prefix only for keys in KVC and all kind of dictionaries.

Comment (kruperfone) [inline Swift-Base/Constants.swift:20]: We must use some prefix for notifications for autocomplete.
Maybe not exactly "k". I think we must discuss this at Wednesday

**Pull request merged**